data/mappings/keyboard_aliases.hjson

```diff
@@ -513,6 +513,9 @@
     "ropro": {
         "target": "kingly_keys/ropro"
     },
+    "rs40": {
+        "target": "kj_modify/rs40"
+    },
```

Comment:
No idea if these aliases are only meant for backwards compatibility (when a board's files get moved into a new folder) or if they can be used in a "normal" scenario like this too